### DISCUSSION TITLE
Nesting generated schema

### DISCUSSION THREAD

**Discussion by jberry92** [MichalLytek/typegraphql-prisma#102] [Q&A] [answered]
Hello, really loving the package its absolutely amazing and just have one little question. I can see that it generates the schema and puts queries and mutations under those types which is fantastic, what I am wondering if it is possible to move them up a level somehow. 

For example this is what I have now 

![image](https://user-images.githubusercontent.com/15660099/115424389-93491880-a1f6-11eb-86f7-2aab33762237.png)

and what I am trying to achieve is getting something like this generated

![image](https://user-images.githubusercontent.com/15660099/115424688-d3100000-a1f6-11eb-8db4-e21af7e52c7d.png)

The use case is that I am trying to integrate with a federated schema and I want to put all of my Queries under a type to keep them hidden unless someone actually wants to find them (not shown at the top level of the playground). So I think to summarise my question is - is it possible to provide an initial type to nest all generated resolvers/types under? Hopefully my question makes sense!

**Comment by MichalLytek** [comment, marked as answer]
Nested queries and mutations (aka namespaces) are an antipattern in GraphQL and are not supported in TypeGraphQL at all.

**Reply by jberry92** [reply]
Alright I'll look into that and see if the federation server can be changed! Thank you
